Timelines/Timeline_onScroll_Animation_1/style.css

```diff
@@ -0,0 +1,334 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap');
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100vw;
+  height: min-content;
+  overflow-x: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background: var(--primary-color);
+}
+
+:root {
+  --ff-prim: 'Open Sans', sans-serif;
+  --ff-sec: 'Roboto', sans-serif;
+  --ff-sp: 'Inter', sans-serif;
+
+  --primary-color: #f6f9fc;
+  --secondary-color: #ffffff;
+
+  --light-gray: rgb(78, 78, 78);
+
+  --pink-dark: #9251ac;
+  --pink-light: #f2a1ea;
+  --blue-dark: #555ac0;
+  --blue-light: #87bbfe;
+  --green-dark: #24b27e;
+  --green-light: #aff1b6;
+}
+
+header {
```

Comment:
Avoid styling html tags directly, Instead move these styles to a css class

Reply:
Those where just added for user demonstration of the slide animation . No worries i will add classes to them .